frontend/src/metabase/dashboard/actions/tabs.ts

```diff
@@ -80,6 +123,15 @@ export function createNewTab() {
   return createNewTabAction({ tabId });
 }
 
+const duplicateTabAction = createAction<DuplicateTabPayload>(DUPLICATE_TAB);
+
+export function duplicateTab(sourceTabId: DashboardTabId | null) {
+  const newTabId = tempTabId;
+  tempTabId -= 2;
+
+  return duplicateTabAction({ sourceTabId, newTabId });
+}
```

Comment:
Not sure what we're doing with `tempTabId -= 2` here. Is it worth adding a comment?

Reply:
Good callout, there's a comment above in `createNewTab` explaining it there, I'll copy it over